dev-docs/bidders/rubicon.md

```diff
@@ -18,6 +18,7 @@ pbs: true
 pbs_app_supported: true
 fpd_supported: true
 gvl_id: 52
+multiformat_supported: will-bid-on-one
```

Comment:
@bretg not sure if I remembered correctly what the rubicon behaviour is :smile:

Reply:
thanks @muuki88 - will-bid-on-one is correct at this point. We're working towards better support and will update this when released.